src/domain/service.ts

```diff
@@ -105,7 +105,12 @@ export async function getApplicationAssetsAsStream(
   const appDoc = await findApplication(appId, identity);
   const appDocObj = appDoc.toObject() as Application;
 
-  if (appDocObj.state !== 'REVIEW' && appDocObj.state !== 'APPROVED') {
+  if (
+    appDocObj.state !== 'REVIEW' &&
+    appDocObj.state !== 'APPROVED' &&
+    // can download assets if app is CLOSED and but was APPROVED (has expiresAt key).
```

Comment:
If there's a better way to check for this let me know! I'm not sure how this would work for applications that are renewing. But just want to prevent attempts to download assets for applications that never got past the review stage.

Reply:
maybe we do store approvedAtUtc date ?

Reply:
yep!